charts/trino/README.md

```diff
@@ -1,119 +1,120 @@
-| `worker.config.query.maxMemoryPerNode` |  | `"1GB"` |
-| `worker.additionalJVMConfig` |  | `[]` |
-| `worker.additionalExposedPorts` |  | `{}` |
-| `worker.resources` |  | `{}` |
-| `worker.livenessProbe` |  | `{}` |
-| `worker.readinessProbe` |  | `{}` |
-| `worker.lifecycle` |  | `{}` |
-| `worker.terminationGracePeriodSeconds` |  | `30` |
-| `worker.nodeSelector` |  | `{}` |
-| `worker.tolerations` |  | `[]` |
-| `worker.affinity` |  | `{}` |
-| `worker.additionalConfigFiles` |  | `{}` |
-| `worker.additionalVolumes` | One or more additional volume mounts to add to all workers. | `[]` |
-| `worker.additionalVolumeMounts` | One or more additional volume mounts to add to all workers. | `[]` |
-| `worker.annotations` |  | `{}` |
-| `worker.labels` |  | `{}` |
-| `worker.secretMounts` |  | `[]` |
-| `kafka.mountPath` |  | `"/etc/trino/schemas"` |
-| `kafka.tableDescriptions` |  | `{}` |
-| `commonLabels` | Labels that get applied to every resource's metadata | `{}` |
-| `ingress.enabled` |  | `false` |
-| `ingress.className` |  | `""` |
-| `ingress.annotations` |  | `{}` |
-| `ingress.hosts` |  | `[]` |
-| `ingress.tls` |  | `[]` |
+## Values
 
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| accessControl | object | `{}` |  |
+| additionalCatalogs | object | `{}` |  |
+| additionalConfigProperties | object | `{}` |  |
+| additionalExchangeManagerProperties | object | `{}` |  |
+| additionalLogProperties | object | `{}` |  |
+| additionalNodeProperties | object | `{}` |  |
+| auth | object | `{}` |  |
+| commonLabels | object | `{}` |  |
+| coordinator.additionalConfigFiles | object | `{}` |  |
+| coordinator.additionalExposedPorts | object | `{}` |  |
+| coordinator.additionalJVMConfig | list | `[]` |  |
+| coordinator.additionalVolumeMounts | list | `[]` |  |
+| coordinator.additionalVolumes | list | `[]` |  |
+| coordinator.affinity | object | `{}` |  |
+| coordinator.annotations | object | `{}` |  |
+| coordinator.config.memory.heapHeadroomPerNode | string | `""` |  |
+| coordinator.config.query.maxMemoryPerNode | string | `"1GB"` |  |
+| coordinator.jvm.gcMethod.g1.heapRegionSize | string | `"32M"` |  |
+| coordinator.jvm.gcMethod.type | string | `"UseG1GC"` |  |
+| coordinator.jvm.maxHeapSize | string | `"8G"` |  |
+| coordinator.labels | object | `{}` |  |
+| coordinator.lifecycle | object | `{}` |  |
+| coordinator.livenessProbe | object | `{}` |  |
+| coordinator.nodeSelector | object | `{}` |  |
+| coordinator.readinessProbe | object | `{}` |  |
+| coordinator.resources | object | `{}` |  |
+| coordinator.secretMounts | list | `[]` |  |
+| coordinator.terminationGracePeriodSeconds | int | `30` |  |
+| coordinator.tolerations | list | `[]` |  |
+| env | list | `[]` |  |
+| envFrom | list | `[]` |  |
+| eventListenerProperties | object | `{}` |  |
+| image.digest | string | `""` |  |
+| image.pullPolicy | string | `"IfNotPresent"` |  |
+| image.registry | string | `""` |  |
```

Comment:
This is an example where description is missing.

Reply:
Looks like `helm-docs` process descriptions that start with a double dash. It's documented here: https://github.com/norwoodj/helm-docs?tab=readme-ov-file#valuesyaml-metadata

Weirdly, I don't see a way to disable/configure this. I don't really want to change all descriptions unless there's no other option. In that case, maybe it'll be a good chance to review them.

Reply:
There are only a few there at the moment .. I have a task on my backlog to clean this up. All of them should have some description in my opinion .. and then the docs site we add should build it up more to create reasonable examples for different use cases...

Reply:
Yeah, that's hardcoded in the tool: https://github.com/norwoodj/helm-docs/blob/4dc6510f8eafae294891bead0a4f4df5e49ad1c4/pkg/helm/chart_info.go#L18

I'll review all existing descriptions and let's see how it'll look.

Reply:
Another question - there's a flag to select sort order, do we want alphabetical or the order from `values.yaml`?